app/page.js

```diff
@@ -70,66 +103,62 @@ export default function Home() {
       payload: "meal",
     });
   }
+
+  function removeUpdatingOrder() {
+    localStorage.removeItem('updatingOrder');
+  }
+  
   return (
     <main className="  flex flex-col ">
       {modalVisible && (
         <div className="fixed left-0 top-0 z-50 flex h-full w-full items-center justify-center bg-black bg-opacity-50 backdrop-blur-sm">
-          <div className="  flex  w-1/2 bg-white   text-dark-purple shadow-md">
-            {previousCustomer ? (
-              <div className=" flex w-2/3 flex-col  gap-4  p-20  ">
-                <h1 className="text-xl">YOUR ORDER</h1>
-                <div className="flex flex-col gap-2 border-b border-dark-purple pb-4">
-                  <p className="text-sm">Date:</p>
-                  <p className=" text-xs">{previousCustomer.date.date}</p>
-                </div>
-                <div className="flex flex-col gap-2 border-b border-dark-purple pb-4">
-                  <p className="text-sm">Time:</p>
-                  <p className="text-xs">{previousCustomer.date.time}</p>
-                </div>
-                <div className="flex flex-col gap-2 border-b border-dark-purple pb-4">
-                  <p className="text-sm">Guests:</p>
-                  <p className="text-xs"> {previousCustomer.people}</p>
-                </div>
-                <div className="flex flex-col gap-2 border-b border-dark-purple pb-4">
-                  <p className="text-sm">Meal: </p>
-                  <p className="text-xs">{previousCustomer.meal}</p>
-                </div>
-                <div className="flex flex-col gap-2  pb-6">
-                  <p className="text-sm">Drinks: </p>
-                  <div className="text-xs">
-                    {previousCustomer.drinks.map((drink, index) => (
-                      <p key={index}>{drink}</p>
-                    ))}
+          <div className="flex w-1/2 bg-white text-dark-purple shadow-md">
+            {previousCustomer && previousCustomer.length ? (
+              <div className="flex w-2/3 flex-col gap-4 p-20">
+                <h1 className="text-xl">YOUR ORDERS</h1>
+                {previousCustomer.map((customer) => (
```

Comment:
When there are multiple orders, they do not fit to the screen. Maybe you could display them in a different way?  Also, as a user, I would expect to see the whole order and not only the meals, so that I could see what I ordered before I click to update it.